htlcswitch/link.go

```diff
@@ -2376,7 +2376,14 @@ func (l *channelLink) canSendHtlc(policy ForwardingPolicy,
 	}
 
 	// Check to see if there is enough balance in this channel.
-	if amt > l.Bandwidth() {
+	bandwidth := l.Bandwidth()
+	if amt > bandwidth {
+		// Log a warning because pathfinding should have avoided this
```

Comment:
Won't this also log for forwards that attempt to go down a link with insufficient bandwidth, since `canSendHtlc` is called by `CheckForwardHtlc`? Not against logging per-say, but the comment might need updating?

Reply:
Hmm good point. Will look into it.

Reply:
Removed the commit